### PR TITLE
Lsl copy phrase bug

### PR DESCRIPTION
**Pull request by lawhead**
* Updated copy_phrase to use the acquisition public properties; this fixes a bug where fs was not being set since it was getting a value of undefined; this was reported in Pivotal Tracker bug #158405692.

* Updated process_data_for_decision method to ensure that output np.array has dtype of float. All downstream code is expecting a float array parameter. This is the default return value when all channel data are floats, however, when we write markers directly to LSL, these values are strings and need to be explicitly converted to sentinel float values.

### CODE REVIEW

File: bcipy/helpers/bci_task_related.py [hunk 1]
```diff
@@ -122,9 +122,20 @@ def process_data_for_decision(sequence_timing, daq):
             if len(raw_data) < data_limit:
                 raise Exception("Not enough data received")
 
-        # Take only the sensor data from raw data and transpose it
-        raw_data = np.array([np.array(raw_data[i][0]) for i in
-                             range(len(raw_data))]).transpose()
+
+        def float_val(col):
```
Comment (tab-cmd) [inline bcipy/helpers/bci_task_related.py]: This can be moved outside of this function.

File: bcipy/helpers/bci_task_related.py [hunk 2]
```diff
@@ -122,9 +122,20 @@ def process_data_for_decision(sequence_timing, daq):
...
+            """Convert marker data to float values so we can put them in a
+            typed np.array. The marker column has type float if it has a 0.0
+            value, and would only have type str for a marker value."""
+            if type(col) is str:
```
Comment (tab-cmd) [inline bcipy/helpers/bci_task_related.py]: use isinstance. 

```
if isinstance(col, str):
```

File: bcipy/helpers/bci_task_related.py (hunk 1 above)
Comment (tab-cmd) [inline bcipy/helpers/bci_task_related.py]: This can be of any type correct? Or at least str/float? Add typing to it

```
from typing import Any
```

**Review by tab-cmd — APPROVED** [review]
Some small things to change

File: bcipy/helpers/bci_task_related.py (hunk 2 above)
Comment (lawhead) [reply bcipy/helpers/bci_task_related.py]: Thanks, good suggestions.

**Pull request merged**